src/lib.rs

```diff
@@ -277,14 +279,12 @@ mod tests {
     #[cfg(feature = "vault_0.6.1")]
     fn it_can_list_things() {
         let c = Client::new(HOST, TOKEN).unwrap();
-        let _ =
-            c.create_token(&client::TokenOptions::default()
-                    .ttl(client::VaultDuration::minutes(1)))
-                .unwrap();
-        let _ =
-            c.create_token(&client::TokenOptions::default()
-                    .ttl(client::VaultDuration::minutes(1)))
-                .unwrap();
+        let _ = c.create_token(&client::TokenOptions::default().ttl(
+            client::VaultDuration::minutes(1),
+        )).unwrap();
+        let _ = c.create_token(&client::TokenOptions::default().ttl(
```

Comment:
We shouldn't need to create this token twice

Reply:
Oh whoops.  I think that was a mistake